web/.c8rc.json

```diff
@@ -7,7 +7,17 @@
   "reporter": ["text", "text-summary"],
   "reports-dir": "build/coverage",
   "src": [
-    "**/src/",
+    "src/**",
     "../common/**/src"
+  ],
+  "include": [
+    "src/**"
+  ],
+  "exclude": [
+    "src/test/**",
+    "**/src/tests/**",
+    "**/build/**",
+    "src/tools/**",
+    "../node_modules/*"
```

Comment:
Is this needed? Doesn't it get excluded because it's not in `include`?

Reply:
Example path: [web/src/engine/predictive-text/worker-thread/src/tests/mocha/cases/auto-correct.js](https://github.com/keymanapp/keyman/blob/be712596c1bc363dce4c734e3ff8ec50c69dc670/web/src/engine/predictive-text/worker-thread/src/tests/mocha/cases/auto-correct.js).

This path _is_ within include... it just matches a certain part of it later down the line.

All predictive-text tests have their definitions within their child project folders at present, I believe.